app/services/queue_health_check.rb

```diff
@@ -0,0 +1,14 @@
+class QueueHealthCheck
+  def perform
+    health_check_jobs.each do |job|
+      $redis.set(job.downcase, 'dead')
+      Object.const_get("HealthCheck::#{job}QueueJob").perform_later
```

Comment:
I think that making a separate class for each queue makes it easy to forget to add one.

Based on the [Active Job Basics Rails Guide](http://edgeguides.rubyonrails.org/active_job_basics.html#queues) I believe we can refactor this to be one job class with an argument for which queue to check:

From the guide:

> If you want more control on what queue a job will be run you can pass a :queue option to #set:
>
> ```ruby
> MyJob.set(queue: :another_queue).perform_later(record)
> ```

Which means I think we could do something like:

```ruby
HealthCheck::CheckerJob.set(queue: 'analytics').perform_later('analytics')

class CheckerJob
  def perform(queue_name)
    $redis.set(queue_name, 'alive') # TODO: use a timestamp
  end
end
```

WDYT?